package.json

```diff
@@ -85,7 +84,7 @@
     "express": "^4.17.1",
     "fetch-mock": "^9.10.6",
     "formik": "^2.2.9",
-    "history": "^5.0.0",
+    "history": "^4.10.1",
```

Comment:
We use react-router v5 which uses history v4 internally. We should be using history 4, too. https://github.com/remix-run/history/issues/804#issuecomment-899124838

This also solves some typescript issues we've had.